.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  industrial_ci:
+    name: ROS2 (${{ matrix.ros_repo }})  
+    strategy:
+      fail-fast: false
+      matrix:
+        env:
+          - {ROS_DISTRO: dashing, ROS_REPO: ros, ABICHECK_URL: "github:ros-drivers/openni2_camera#ros2"}
```

Comment:
Not sure why I added dashing...this config was a copy from another repo. Oh well.